ui/app/templates/release_scheduled_change_modal.html

```diff
@@ -6,14 +6,21 @@ <h3 class="modal-title" ng-show="sc.name && sc.when">Edit Scheduled Change for {
 <div class="modal-body">
   <form role="form">
     <h4>Scheduled Change Details</h4>
-
       <div class="row">
+        <div class="col-md-8 col-md-offset-2">
+            <div class="help-block">
+                <div ng-hide="hideAsapInfo" class="alert alert-info" auto-hide="3000">
+                    <a href="#" class="close" arial-label="close" ng-click="hideAsapInfo=true">&times;</a>
+                    <strong>Info !</strong> Use ASAP to make changes as soon as possible
+            </div>
+        </div>
       <div class="col-md-8 col-md-offset-2">
         <label for="id_when">When (local time)</label>
+      </div>
         <div class="form-group dropdown" is-open="calendar_is_open" ng-class="{'has-error': errors.when}">
           <div class="input-group">
             <span class="input-group-btn">
-              <label class="btn btn-default" ng-click="toggleAutoTime()" id="btn__auto-time"><input type="checkbox" ng-show="false" ng-model="auto_time" auto-complete="off">ASAP</label>
+              <label class="btn btn-primary" ng-click="toggleAutoTime()" id="btn__auto-time"><input type="checkbox" ng-show="false" ng-model="auto_time" auto-complete="off">ASAP</label>
```

Comment:
This styling change doesn't really improve anything...the button still has an unclear state. I'm not exactly sure what it should look like though...

Reply:
@mozbhearsum  Hmm, will us adding some color and a border to it do the trick ?
i.e
<img width="255" alt="screen shot 2017-10-19 at 11 51 10 am" src="https://user-images.githubusercontent.com/11579108/31762852-31d85ed4-b4c5-11e7-8234-d09765f1852c.png">

Reply:
The "after" screenshot looks like the button has already been depressed still.

Reply:
@mozbhearsum What about adding a little elevation to it, to something like :

<img width="359" alt="screen shot 2017-10-23 at 10 30 55 am" src="https://user-images.githubusercontent.com/11579108/31877021-95528154-b7dd-11e7-8034-ab39a51aadd9.png">

Reply:
@kmoir - what do you think? If I remember correctly, you found the current styling of the ASAP button a bit unclear.